packages/import-manager/test/unitTests/test-importsA.js

```diff
@@ -1,46 +1,40 @@
 // Copyright (C) 2019 Agoric, under Apache License 2.0
 
 import '@agoric/install-ses';
-import { test } from 'tape-promise/tape';
+import test from 'ava';
 import { makeGoodImportManager } from './goodImports';
 
 test('import num is not empty', t => {
   const importer = makeGoodImportManager();
   const name = 'numIsEmpty';
   const emptyFn = importer[name];
-  t.notOk(emptyFn(30));
-  t.end();
+  t.falsy(emptyFn(30));
 });
 
 test('import num is empty', t => {
   const importer = makeGoodImportManager();
   const name = 'numIsEmpty';
   const emptyFn = importer[name];
-  t.assert(emptyFn(0));
-  t.end();
+  t.truthy(emptyFn(0));
 });
 
 test('import listIsEmpty (false)', t => {
   const importer = makeGoodImportManager();
   const op = 'listIsEmpty';
-  t.notok(importer[op]([20]));
-  t.end();
+  t.falsy(importer[op]([20]));
 });
 
 test('import listIsEmpty (true)', t => {
   const importer = makeGoodImportManager();
   const op = 'listIsEmpty';
-  t.assert(importer[op]([]));
-  t.end();
+  t.truthy(importer[op]([]));
 });
 
 // TODO: This test throws because `lookupImport` does not exist. This
 // test needs to be fixed.
 test.skip('import not found', t => {
   const importer = makeGoodImportManager();
-  t.throws(
-    () => importer.lookupImport('emptyPixel'),
-    /There is no entry for "c"./,
-  );
-  t.end();
+  t.throws(() => importer.lookupImport('emptyPixel'), {
```

Comment:
I'd put the reporter on it's own line, but maybe this is a `prettier` thing?

Reply:
yeah, I typed it on its own line, then `prettier` rewrote it into that ugly thing